src/store/builder/index.ts

```diff
@@ -350,6 +350,9 @@ export function generateConnectionLayoutKeyFor(source: IBlock, target: IBlock):
     source.label,
     target.label,
 
+    // states that would affect block heights
+    source?.vendor_metadata?.floip?.ui_metadata?.should_show_block_tool_bar,
```

Comment:
This is what we need to do for now to trigger the connection `reposition()`, by updating which key should be considered ;)

Reply:
let's rename `tool_bar` to one-word `toolbar`
(and the same for camelCase and PascalCase in the code)